app/javascript/components/app.css

```diff
@@ -1,16 +1,16 @@
 .greeting {
```

Comment:
- [ ] You app is rendering a blank screen when the `Click [here](http://127.0.0.1:3000/greeting) to see a greeting!` link is invoked. The _greeting_ action loads an empty screen as you don't have sample random data to be loaded. Kindly ensure to populate the _seed file_ with sample greeting messages so that the sample message would be loaded from the databases when greeting action is called.

Reply:
This issue is well resolved! ✔️